proto/prysm/v1alpha1/beacon_chain.proto

```diff
@@ -467,7 +467,13 @@ message BeaconBlockContainer {
 
     // Representing a blinded Electra block.
     SignedBlindedBeaconBlockElectra blinded_electra_block = 12;
-  }
+
+    // Representing a Fulu block.
+    SignedBeaconBlockFulu fulu_block = 13;
+
+    // Representing a blinded Fulu block.
+    SignedBlindedBeaconBlockFulu blinded_fulu_block = 14;
+    }
```

Comment:
I don't see this in the nits doc. Is this related?

Reply:
no longer in the PR